locales/en-US/app.ftl

```diff
@@ -1,23 +1,20 @@
 siteName = Firefox Test Pilot
 
-// Page titles, put in the <title> HTML tag.
-[[pageTitle]]
+## Page titles, put in the <title> HTML tag.
```

Comment:
drive-by nit, put-put-put. `shown as title of the html page` is less techy, and doesn't sound like you should add <title> around your strings.